Firmware-ESP/platformio.ini

```
@@ -18,6 +18,14 @@ framework = arduino
 monitor_filters = esp32_exception_decoder
 upload_speed = 921600
 
+[env:esp32_ota]
+; extends the settings from the [env] section for OTA updating
+extends = env
+upload_protocol = espota
+; IP address of the ESP32
+upload_port = 192.168.0.49
```

```suggestion
upload_port =
```
The IP address is specific to your installation. I'd rather leave this empty; platformio will then show an error message that this setting needs to be set, which is OK.

---

Good spot!